### PR TITLE
Add new target CSKY405 and fix BMI088 driver

### PR DESCRIPTION
**Pull request by AlexandrShipovsky**
I have added the configuration file of the board that our company produces. 
I also added the ability to set the address of the BMP388 barometer, since in my case it is necessary.
In addition, the BMI088 sensor refused to work properly with the supplied driver. It seems there was an error in the alignment of the axes of the gyroscope and accelerometer. With the current changes, my single sensor board BMI088 is working fine. It would be great if someone else can check this on another board.

### CODE REVIEW

File: src/main/drivers/barometer/barometer_bmp388.c [hunk 1]
```diff
@@ -39,7 +39,12 @@
 
 #if defined(USE_BARO) && (defined(USE_BARO_BMP388) || defined(USE_BARO_SPI_BMP388))
 
+#ifndef BMP388_I2C_ADDR_HW
```
Comment (mmosca) [inline src/main/drivers/barometer/barometer_bmp388.c]: This check is repeated in 2 places. It should probably be in a header somehwere else.

**Comment by mmosca** [comment]
In case you haven't seen it before, this is INAV's new hardware policy:
https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md

If you are already in touch with INAV devs to arrange for test samples, you can ignore this, otherwise please have a look at the policy and get in touch either on Discord here so we can validate the hardware for inclusion.

**Comment by AlexandrShipovsky** [comment]
> Thank you for the PR, In case you haven't seen it before, this is INAV's new hardware policy: https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
> 
> If you are already in touch with INAV devs to arrange for test samples, you can ignore this, otherwise please have a look at the policy and get in touch either on Discord here so we can validate the hardware for inclusion.

I have not seen this policy before. Please let me know who I can send the samples to?

File: src/main/drivers/barometer/barometer_bmp388.c (hunk 1 above)
Comment (AlexandrShipovsky) [reply src/main/drivers/barometer/barometer_bmp388.c]: So it's really strange code, thank you, i fixed it

**Comment by AlexandrShipovsky** [comment]
@mmosca this flight controller was added to the ardupilot, it is a pity that it is not possible to quickly add it to INAV https://github.com/ArduPilot/ardupilot/pull/26667

**Pull request closed without merging**